### PR TITLE
Create the Insights Engine

### PR DESCRIPTION
**Pull request by AlexisMLopez**
Create the system's Insights Engine to predict the medical infrastructure needs in a geographical area by cross-referencing Synthea and Census data.

### CODE REVIEW

File: install/sql/api_views.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS acs.population_change
+(
+    state 			TEXT,
```
Comment (htbryant1256) [inline install/sql/api_views.sql:3]: Fix styling indentation problems

File: install/sql/api_views.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    year_range		TEXT,
+    percent_change	NUMERIC
+);
+
+CREATE TABLE IF NOT EXISTS acs.results
+(
+    year		    TEXT,
```
Comment (htbryant1256) [inline install/sql/api_views.sql:10]: Fix styling indentation problems

File: server/db/queries.js [hunk 3]
```diff
@@ -38,34 +39,207 @@ const getCarePlans = () => {
...
+        text: "SELECT allergenic_patients FROM synthea.peanut_allergy_patients",
+        rowMode: 'array',
+    };
+
+    return client.query(query)
+        .then((results) => {
+            return results.rows[0];
+        });
+};
+
```
Comment (htbryant1256) [inline server/db/queries.js:78]: Remove newline

File: server/db/queries.js [hunk 4]
```diff
@@ -38,34 +39,207 @@ const getCarePlans = () => {
...
+
```
Comment (htbryant1256) [inline server/db/queries.js]: Remove newline

File: server/insights_engine/prediction_algorithms.js [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+            const peanutAllergyResults = await getResults();
+            resolve(peanutAllergyResults);
+        } catch (error) {
+            reject(error);
+        }
+    });
+}
+
+
```
Comment (htbryant1256) [inline server/insights_engine/prediction_algorithms.js]: Remove newline

File: server/insights_engine/prediction_algorithms.js [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        }
+    });
+}
+
+async function calculateInsulin() {
+    return new Promise(async (resolve, reject) => {
+        try {
+            truncateTable();
+
+
```
Comment (htbryant1256) [inline server/insights_engine/prediction_algorithms.js:76]: Remove newline

File: server/insights_engine/prediction_algorithms.js [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+            let populationChange = await getPopulationChange2020_2021();
+            populationChange = parseFloat(populationChange[0]) + 1;
+
+
```
Comment (htbryant1256) [inline server/insights_engine/prediction_algorithms.js:80]: Remove newline

File: server/insights_engine/prediction_algorithms.js [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+            }
+
+            const insulinResults = await getResults();
+            resolve(insulinResults);
+        } catch (error) {
+            reject(error);
+        }
+    });
+}
+
```
Comment (htbryant1256) [inline server/insights_engine/prediction_algorithms.js:110]: Remove newline

**Review by htbryant1256 — CHANGES_REQUESTED** [review]

**Review by Andrew-Wilhelm — APPROVED** [review]

**Review by htbryant1256 — APPROVED** [review]

**Pull request merged**